api/src/test/java/org/pmiops/workbench/db/dao/UserDaoTest.java

```diff
@@ -424,6 +433,68 @@ public void testFindUserByUsernames() {
         .containsExactly(alice, bob);
   }
 
+  @Test
+  public void
+      testFindUsersBetweenCreationTimeWithoutNewUserSurveyOrCode_returnsUsersCreatedBetweenTimes() {
+    Instant creationTimeWindowStart = Instant.parse("2000-01-01T00:00:00.00Z");
+    Instant creationTimeWindowEnd = creationTimeWindowStart.plus(1, ChronoUnit.DAYS);
+
+    // Invalid user near window start
+    userWithCreationTime(Timestamp.from(creationTimeWindowStart.minus(1, ChronoUnit.SECONDS)));
+    DbUser validUserNearWindowStart =
+        userWithCreationTime(Timestamp.from(creationTimeWindowStart.plus(1, ChronoUnit.SECONDS)));
+    DbUser validUserNearWindowEnd =
+        userWithCreationTime(Timestamp.from(creationTimeWindowEnd.minus(1, ChronoUnit.SECONDS)));
+    // Invalid user near window end
+    userWithCreationTime(Timestamp.from(creationTimeWindowEnd.plus(1, ChronoUnit.SECONDS)));
+
+    List<DbUser> users =
+        userDao.findUsersBetweenCreationTimeWithoutNewUserSurveyOrCode(
+            Timestamp.from(creationTimeWindowStart), Timestamp.from(creationTimeWindowEnd));
+    assertThat(users)
+        .containsExactlyElementsIn(
+            ImmutableList.of(validUserNearWindowStart, validUserNearWindowEnd));
+  }
+
+  @Test
+  public void
+      testFindUsersBetweenCreationTimeWithoutNewUserSurveyOrCode_returnsUsersWithoutOneTimeCodes() {
+    Instant creationTimeWindowStart = Instant.parse("2000-01-01T00:00:00.00Z");
```

Comment:
this variable looks arbitrary and hard to connect that with FakeClockConfig. 
Can you static some variable from this class: https://github.com/all-of-us/workbench/blob/e7587f5899df5be5fd2ba254be972cc4e6112205/api/src/test/java/org/pmiops/workbench/FakeClockConfiguration.java#L14?

Reply:
It's arbitrary on purpose, and a complete coincidence that it's the same date as the one used in FakeClockConfig! Using `NOW` would give the false impression that the test depends on the current time.

Instead of using `NOW`, I've switched the tests to use a different time to avoid that confusion.